lighthouse-core/report/html/renderer/report-renderer.js

```diff
@@ -255,10 +267,24 @@ class ReportRenderer {
     reportSection.appendChild(this._renderReportFooter(report));
 
     const reportFragment = this._dom.createFragment();
+
     if (!this._dom.isDevTools()) {
       const topbarDocumentFragment = this._renderReportTopbar(report);
       reportFragment.appendChild(topbarDocumentFragment);
     }
+
+    if (scoreHeader && !this._dom.isDevTools()) {
+      const stickyHeader = this._dom.createElement('div', 'lh-sticky-header');
```

Comment:
we need a test for this, though jsdom (AFAIK) doesn't support scrolling, so it will have to be a limited one. Since we have a nice complete "renders score gauges in this order" test, maybe just a test that `.lh-sticky-header` has child gauges that match the ones in `.lh-scores-header`?

Or something like that

Reply:
> a test that .lh-sticky-header has child gauges that match the ones in .lh-scores-header?

imo that doesn't seem like a useful test, since these gauges are created with the same function.

> jsdom (AFAIK) doesn't support scrolling

https://github.com/jsdom/jsdom/issues/1422#issuecomment-471038750 this makes me think it should be possible to fake it

Reply:
ok it'd be way too much faking, basically all the `getBoundingClientRect` needs to be mocked